server/leader.go

```diff
@@ -188,3 +205,7 @@ func (s *Server) watchLeader() {
 		}
 	}
 }
+
+func (s *Server) leaderCmp() clientv3.Cmp {
+	return clientv3.Compare(clientv3.Value(s.getLeaderPath()), "=", s.leaderValue)
```

Comment:
yes, the string is created before server runs, and will be used as a const value and doesn't change. 
But later I will use protobuf go generate this value because outer client will fetch this value and we 
should use protobuf uniformly.